app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None | str:
+        if "vaccine" not in visitor.keys():
+            raise NotVaccinatedError("Not Vaccinated")
+        today_data = datetime.date.today()
+        date2 = visitor["vaccine"]["expiration_date"]
```

Comment:
please, use descriptive messages. `date2` doesn't say anything about what is stored in the variable